src/features/fireWeather/components/graphs/TempRHGraph.styles.ts

```diff
@@ -1,14 +1,18 @@
 import { makeStyles } from '@material-ui/core/styles'
 
 const currLineColor = 'green'
-export const readingTempDotColor = '#ff6984'
-export const readingRHDotColor = '#6198ff'
-export const modelTempDotColor = '#ff0d39'
+export const readingTempDotColor = '#ff1212'
+export const readingRHDotColor = '#196aff'
+export const modelTempDotColor = '#ff6984'
+export const modelRHDotColor = '#6198ff'
 export const biasModelTempDotColor = '#aa0000'
 export const biasModelRHDotColor = '#0000aa'
-export const modelRHDotColor = '#1467ff'
 export const modelSummaryTempAreaColor = '#ff96aa'
 export const modelSummaryRHAreaColor = '#94b9ff'
+export const highResModelTempDotColor = '#ff2b52'
```

Comment:
Is having different colours for the same metric really a smart move? I would think disallowing the showing HRDPS and GDPS at the same time would make sense? Don't change this right now - I think it's just a good talking point in the demo asking this question.

Reply:
Yeah, this is sort of temporary solution, my vision is having another select input where you can switch between HRDPS and GDPS. And use the same colours for model related values